mlir/utils/jenkins/Jenkinsfile

```diff
@@ -267,12 +269,10 @@ pipeline {
                             }
                         }
                         stage("Stash MIOpen with libMLIRMIOpen") {
-                            when { allOf {
-                                equals expected: true, actual: params.nightly;
-                                anyOf {
-                                equals expected: true, actual: params.perfTest;
-                                equals expected: true, actual: params.canXdlops;
-                            } } }
+                            when { anyOf {
+                                    equals expected: true, actual: params.nightly;
+                                    equals expected: true, actual: params.weekly;
+                            }}
```

Comment:
Same deal - let's make sure we can run individual parts of nightly

Reply:
> I'm not entirely understanding how tuning will interact with the rest of nightly CI - ex. with the performance reports, especially since the important bit seems to be commented out. Can you clarify?

The tuning results will be used by the performance reports in nightly CI. This will be in another PR.

Reply:
Re the control constants, what I mean is we still want the `perfTest` variable so that we can run just performance tests, or something analogous to it.